lib/BoundaryValueDiffEqMIRK/src/adaptivity.jl

```diff
@@ -50,7 +51,116 @@ Generate new mesh based on the defect.
     n_ = T(0.1) * n
     n_predict = ifelse(abs((n_predict - n)) < n_, round(Int, n + n_), n_predict)
 
-    if r₁ ≤ ρ * r₂
+    if r₁ ≤ ρ * r₃
```

Comment:
I don't know where this comes from, but this is basically wrong, and made mesh refinement always halve mesh(because that redistribution clause never gets triggered!!!)